packages/decoder/lib/decoders.ts

```diff
@@ -79,6 +82,14 @@ export class ProjectDecoder {
     if (!provider) {
       throw new NoProviderError();
     }
+    //check for repeat compilation IDs
+    for (let i = 0; i < compilations.length; i++) {
+      for (let j = i + 1; j < compilations.length; j++) {
+        if (compilations[i].id === compilations[j].id) {
```

Comment:
Will repeats be next to each other? Is it possible for i, and i +2 to share the same id if i+1 does not?

Reply:
No, there's no reason repeats have to be next to each other.  This should find any repeats; why is being next to each other relevant?

Reply:
Oh, I read that wrong. Please disregard, I have no good excuse there. :/